apps/web/src/blocks/FAQBlock/FAQBlock.config.ts

```diff
@@ -6,41 +6,45 @@ const FAQBlock: Block = {
   interfaceName: 'FAQBlockT',
   fields: [
     BlockConfig(),
-    {
-      type: 'text',
-      name: 'title',
-      localized: true,
-      label: 'Title',
-      required: false
-    },
     {
```

Comment:
Wondering if we need a dedicated richtext field for the header content here? We could probably get by just using a markdown block. I could see that being more confusing for CMS editors, though.

Reply:
I think for most cases we should just use the RichText Block but having the header embedded is useful in this case because of the textAlignment I added.